### PR TITLE
schema hide true added to route *

### PR DESCRIPTION
**Pull request by chapuletta**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ function fastifyStatic (fastify, opts, next) {
   if (opts.prefix === undefined) opts.prefix = '/'
   const prefix = opts.prefix[opts.prefix.length - 1] === '/' ? opts.prefix : (opts.prefix + '/')
 
-  fastify.get(prefix + '*', function (req, reply) {
+  fastify.get(prefix + '*', { schema: { hide: true } }, function (req, reply) {
```
Comment (mcollina) [inline index.js]: can you please add a comment here explaining why we are doing this?

**Comment by chapuletta** [comment]
Ok, i do the change for ops, i need to do a pull request in fastify-swagger too for adding the opts schema in fastify-static plugin.

File: index.js (hunk 1 above)
Comment (chapuletta) [reply index.js]: done in the newest changes. If need something else, please advice me.

**Comment by chapuletta** [comment]
@allevo like schemaHide: true in options parameters? instead all the schema structure?

File: test/static.test.js [hunk 2]
```diff
@@ -715,3 +715,119 @@ t.test('with fastify-compress', t => {
...
+  const pluginOptions = {
+    root: path.join(__dirname, '/static'),
+    prefix: '/static/',
+    schemaHide: true
+  }
+
+  const fastify = Fastify()
+  fastify.register(fastifyStatic, pluginOptions)
+
+  fastify.addHook('onRoute', function (routeOptions) {
```
[the remark this answers is not in the document]
Comment (chapuletta) [reply test/static.test.js]: done

File: test/static.test.js [hunk 3]
```diff
@@ -715,3 +715,119 @@ t.test('with fastify-compress', t => {
...
+  const pluginOptions = {
+    root: path.join(__dirname, '/static'),
+    prefix: '/static/',
+    schemaHide: false
+  }
+
+  const fastify = Fastify()
+  fastify.register(fastifyStatic, pluginOptions)
+
+  fastify.addHook('onRoute', function (routeOptions) {
```
[the remark this answers is not in the document]
Comment (chapuletta) [reply test/static.test.js]: done

File: test/static.test.js [hunk 4]
```diff
@@ -715,3 +715,119 @@ t.test('with fastify-compress', t => {
...
+
+  const pluginOptions = {
+    root: path.join(__dirname, '/static'),
+    prefix: '/static/'
+  }
+
+  const fastify = Fastify()
+  fastify.register(fastifyStatic, pluginOptions)
+
+  fastify.addHook('onRoute', function (routeOptions) {
```
[the remark this answers is not in the document]
Comment (chapuletta) [reply test/static.test.js]: done

**Comment by allevo** [comment]
Thanks!

**Pull request merged**